src/delaunay.c

```diff
@@ -345,17 +338,31 @@ static int get_extended_basis(double basis[4][3], const double lattice[3][3],
     return lattice_rank;
 }
 
-/* For Monocli/Oblique, unique axis is aperiodic axis, j and k are periodic */
-/* For Monocli/Rectang, k is aperiodic axis, unique axis and j are periodic */
-/* j and k are delaunay reduced, which can be incomplete for Monocli/Rectang */
-static int delaunay_reduce_2D(double red_lattice[3][3],
-                              const double lattice[3][3], const int unique_axis,
-                              const int aperiodic_axis, const double symprec) {
+// @brief Delaunay reduction for monoclinic/oblique or monoclinic/rectangular
+// @param[out] red_lattice
+// @param[in] lattice
+// @param[in] unique_axis
+//            Two-fold axis or mirror-plane-perpendicular axis
+// @param[in] aperiodic_axis
+// @param[in] symprec
+// @note For Monoclinic/oblique, the unique axis is also the aperiodic axis.
+//       Axes are {j, k, unique_axis(=aperiodic_axis)}.
+//       For Monoclinic/rectangular, axes are {unique_axis, j,
+//       k(=aperiodic_axis)}. j and k are delaunay reduced, which can be
+//       incomplete for Monoclinic/Rectangular
+int del_layer_delaunay_reduce_2D(double red_lattice[3][3],
```

Comment:
If you don't plan at exposing or accessing the function via `extern`, make the function `static`. I.e. if you only use it within the same file

Reply:
@LecrisUT, this function is used in `spacegroup.c`, so I think that without making it `static` is fine. But is there any other point? I ask because I don't see what you mean about 

> If you don't plan at exposing or accessing the function via extern

Reply:
Git was reporting that this function was a new one, but after some search, it's just moved. But in that case:
- move the documentation to header file
- synchronize the variable name if possible between source and header
- remove const int while you're editing that section/file

About
> If you don't plan at exposing or accessing the function via extern

I mean anything that is not in a header file should me marked static